tox.ini

```diff
@@ -11,6 +11,23 @@ deps =
     pillow2.0: Pillow<=2.0.9999
     pillow2.2: Pillow<=2.2.9999
     pillow2.4: Pillow<=2.4.9999
+    pillow2.x: Pillow<3.0.0
+    pillow3.0.0: Pillow==3.0.0
+    pillow3.1.1: Pillow==3.1.1
+    pillow3.3.0: Pillow==3.3.0
```

Comment:
Is there any reason for each patch version? I think testing with each major version (ex. pillow6.x) is enough today. So it's time to drop test patterns for 2.0, 2.2 and 2.4.

Reply:
In addition, we need to add test pattern to `.travis.yml` to ensure these cases passed.

Reply:
As described in commit log, these patch versions initially caused behavior change, and testing with them can certainly ensure reproduction of (expected) failure if degraded. Therefore, I added them in order to avoid overlooking degrade in the future by complicated reason.

So, I'm OK for dropping them, if these are too nervous :-)

Then, is testing with 2.x, 3.x, 4.x, 5.x, and 6.x enough ?

And sorry, I overlooked about `.travis.yml`. I'll also revise this commit for it.

Reply:
I pushed amended changeset, which does below:

- drop test patterns for 2.0, 2.2 and 2.4
- add test patterns to travis.yml